### PR TITLE
Add Fluid Attacks' Scanner

### PR DESCRIPTION
**Pull request by kamadorueda**

### CODE REVIEW

File: _data/tools.json [hunk 1]
```diff
@@ -1295,7 +1304,7 @@
       "note": null,
       "type": "DAST"
    },
-  {
+   {
```
Comment (kingthorin) [inline _data/tools.json:1307]: Please revert this change

File: _data/tools.json [hunk 2]
```diff
@@ -989,6 +989,15 @@
       "note": null,
       "type": "SAST"
    },
+   {
+      "title": "Fluid Attack's Scanner",
+      "url": "https://docs.fluidattacks.com/machine/scanner",
+      "owner": "Fluid Attacks",
+      "license": "Open Source",
+      "platforms": null,
+      "note": "SAST, DAST and SCA vulnerability detection tool with perfect OWASP Benchmark score.",
```
Comment (kingthorin) [inline _data/tools.json:998]: @davewichers are you aware of this? Is the claim supportable?

Comment (kamadorueda) [reply _data/tools.json:998]: - Code is open source and you can reproduce results as explained here: https://docs.fluidattacks.com/machine/scanner/reproducibility
- @davewichers is aware: https://github.com/OWASP/Benchmark/pull/146#issuecomment-825996304
  but has no time for the project I guess

Comment (kamadorueda) [reply _data/tools.json:998]: @kingthorin Is this enough proof or should I remove the claim?

We have nothing to hide anyway, code is there: https://gitlab.com/fluidattacks/product/-/tree/master/skims

Also instructions to reproduce results, etc, etc

Comment (kingthorin) [reply _data/tools.json:998]: Thanks!

**Review by kingthorin — APPROVED** [review]

File: _data/tools.json (hunk 2 above)
Comment (kingthorin) [reply _data/tools.json:998]: I’m happy with that background info 👍

Comment (kamadorueda) [reply _data/tools.json:998]: Thanks to you for the peer-review 👍🏻
